index.html

```diff
@@ -123,20 +133,26 @@ <h4 class="panel-title">
                         Save password in your browser
                     </label>
                   </div>
+                  <div class="checkbox">
+                    <label class="control-label" for="ssl">
+                      <input type="checkbox" id="ssl" ng-model="settings.ssl">
+                      Encryption. <strong>Strongly recommended!</strong> Need help? Check below.
+                    </label>
+                  </div>
```

Comment:
Why reorder the inputs like that? It's weird to have save password -> encryption -> autoconnect when autoconnect depends on save password

Reply:
Agree but I didn't specifically reorder. I initially added it to the bottom and then moved the input to the top.. Better?